docs/stellar-core_example.cfg

```diff
@@ -236,10 +236,10 @@ BUCKETLIST_DB_INDEX_CUTOFF = 20
 # this value is ingnored and indexes are never persisted.
 BUCKETLIST_DB_PERSIST_INDEX = true
 
-# EXPERIMENTAL_BACKGROUND_EVICTION_SCAN (bool) default false
+# BACKGROUND_EVICTION_SCAN (bool) default true
 # Determines whether eviction scans occur in the background thread. Requires
-# that EXPERIMENTAL_BACKGROUND_EVICTION_SCAN is set to true.
-EXPERIMENTAL_BACKGROUND_EVICTION_SCAN = false
+# that USE_DEPRECATED_SQL is set to false.
```

Comment:
note that this will break nodes that use EXPERIMENTAL_BACKGROUND_EVICTION_SCAN right now (so at least our infra), we should remember to remove that flag before releasing these changes.

Reply:
Yeah, since it's experimental we are probably the only org using these flags, but we'll need to make sure to update the toml files on release accordingly.

Reply:
sg, I updated the name of your PR to remind us about this when putting together release notes for 22.0.0